src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int COLOR_NUMBER = 5;
+    static final int RANDOM_NUMBER = 100;
```

Comment:
```suggestion
    private static final int COLOR_NUMBER = 5;
    private static final int RANDOM_NUMBER = 100;
```
remember about access modifiers